src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -136,7 +148,12 @@ def __call__(
 
         image = (image / 2 + 0.5).clamp(0, 1)
         image = image.cpu().permute(0, 2, 3, 1).numpy()
+
+        # run safety checker
+        safety_cheker_input = self.feature_extractor(self.numpy_to_pil(image), return_tensors="pt").to(torch_device)
+        image, has_nsfw_concept = self.safety_checker(images=image, clip_input=safety_cheker_input.pixel_values)
+
```

Comment:
`safety_checker` will replace nsfw images (if detected) in `image` with a black image.